### ISSUE TITLE
`display` rule does not output in CSS inside a media-query with `float` rule

### ISSUE THREAD

**Issue by phoenixenero** [sass/libsass#1549]
I have no idea why this is happening but here:

```css
.primary-menu__dropdown-toggle {
	// ...

	@media ( max-width: 600px ) {
		float: right;
		display: block;
	}

	// ...
}
```

Expected output:

```css
@media ( max-width: 600px ) {
.primary-menu__dropdown-toggle {
		float: right;
		display: block;
	}
}
```

Output that i got:

```css
@media ( max-width: 600px ) {
.primary-menu__dropdown-toggle {
		float: right;
	}
}
```

Using gulp-sass 2.0.4

**Comment by saper** [comment]
While not identical, here is what I got from libsass git master (681b5113fa74f2cc6117f142d9bdd56dfac494da):

```
@media (max-width: 600px) {
  .primary-menu__dropdown-toggle {
    float: right;
    display: block; } }
```

So this will be fixed in the next release.

**Comment by xzyfer** [comment]
This is fixed in 3.3. You can follow it's release progress by subscribing to https://github.com/sass/libsass/issues/1453
